### PR TITLE
Hw wallet

### PR DESCRIPTION
**Pull request by b-pmcg**

### CODE REVIEW

File: hooks/useHardwareWallet.js [hunk 1]
```diff
@@ -104,7 +104,7 @@ export function useLedger({ onAccountChosen }) {
         }
       });
     },
-    [show]
+    [accountSelection, show]
```
Comment (b-pmcg) [inline hooks/useHardwareWallet.js]: @tyler17 I traced the null `maker` error to `onAccountChosen` in `WalletManager.js`. Adding this dependency gives us access to `maker` in that function & now ledger seems to be working. I wasn't sure if there was more you wanted to do with this branch, so I didn't merge it in yet.

Comment (tyler17) [reply hooks/useHardwareWallet.js]: @b-pmcg Great, thanks!  If everything looks good, we can merge, I was planning on just testing everything a little more.

**Comment by levity** [comment]
I'll take a look at this today. Please don't merge until I give feedback

**Comment by levity** [comment]
> I'll take a look at this today. Please don't merge until I give feedback

Well, I don't have access to my hardware wallets today, so the feedback I can provide is limited. So I'll defer to @tyler17 on the final call here. Merging with Ledger disabled if Trezor's working well is certainly an option.

**Comment by tyler17** [comment]
With trezor, I'm still getting the issue where I'm asked to export my addresses twice, once before selecting addresses, and once after, so will hold off merging with trezor tonight.  Will debug tomorrow. Everything else with trezor seems to be working fine.

**Comment by b-pmcg** [comment]
@tyler17 are you still unable to get ledger balances loading? It works for me at this deployment here: https://migrate-dashboard-git-hw-wallet.mkr-js-prod.now.sh/

**Comment by tyler17** [comment]
@b-pmcg Oh, it works for me on that deployment, but not when I run this branch locally...any idea why?

**Comment by levity** [comment]
> @b-pmcg Oh, it works for me on that deployment, but not when I run this branch locally...any idea why? (same thing happens when I run mcd-cdp-portal locally, but oasis.app works)

It's checking for HTTPS.

**Comment by b-pmcg** [comment]
yeah, ledger complains if you're not using https, I haven't figured out a good way to get https working with zeit/now locally, CRA lets you do `HTTPS=true`. For development I have been mocking the array of addresses returned to the front end.

**Comment by levity** [comment]
Maybe one of these will do the trick: https://www.npmjs.com/search?q=local-ssl-proxy

**Pull request merged**